config/config.toml.example

```diff
@@ -41,6 +41,9 @@ oom-action = "log"
 # Enable coprocessor streaming.
 enable-streaming = false
 
+# Set system variable 'lower_case_table_names'
```

Comment:
Add a comment here for the usage. Could we do not put it here and use the default value in config.go?

Reply:
The config_test.go will check the value in config.toml.example is the same as cfg.LowerCaseTableNames, so we need to put an example here.

Reply:
Got it!